Quadriga/src/main/java/edu/asu/spring/quadriga/passthroughproject/constants/Constants.java

```diff
@@ -0,0 +1,19 @@
+package edu.asu.spring.quadriga.passthroughproject.constants;
+
+public class Constants {
+
+    public static final String VOGONWEB_NETWORK_NAME = "VogenWeb_Details";
+
+    // Constants related to external project
+    public static final String DEFAULT_PROJECT_ID = "external_project_id";
+    public static final String DEFAULT_PROJECT_NAME = "external_project";
+    public static final String VOGONWEB_SENDER = "VogonWeb";
+    public static final String DEFAULT_VOGONWEB_USER_NAME = "VogonWebUser";
+    public static final String DEFAULT_VOGONWEB_USER_ID = "VogonWebUserId";
+    public static final String DEFAULT_PROJECT_DESCRIPTION = "external_project_description";
+
+    // Constants related to external workspace
+    public static final String DEFAULT_WORKSPACE_ID = "external_workspace_id";
+    public static final String DEFAULT_WORKSPACE_NAME = "external_workspace";
+    public static final String DEFAULT_WORKSPACE_DESCRIPTION = "external_workspace_description";
```

Comment:
Let's give those constants, VogonWeb independent strings (no "vogonWeb" in the strings).